# --- T2-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by scripts/Create-CopyPatch.
# 
# T2 SDE: package/*/zstd/hotfix.patch.alpha
# Copyright (C) 2021 The T2 SDE Project
# 
# More information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- T2-COPYRIGHT-NOTE-END ---

--- zstd-1.4.5/programs/util.c.vanilla	2020-10-04 17:20:31.861471356 +0200
+++ zstd-1.4.5/programs/util.c	2020-10-04 17:20:05.057471500 +0200
@@ -20,6 +20,7 @@
 #include <stdlib.h>     /* malloc, realloc, free */
 #include <stdio.h>      /* fprintf */
 #include <time.h>       /* clock_t, clock, CLOCKS_PER_SEC, nanosleep */
+#include <utime.h>
 #include <errno.h>
 #include <assert.h>
 
